examples/message_passing/after.cpp

```diff
@@ -0,0 +1,57 @@
+// This example shows how to use caf::after
+
+#include "caf/all.hpp"
+
+#include <chrono>
+#include <iostream>
+#include <random>
+
+using std::cout;
+using std::endl;
+
+using namespace caf;
+
+// Sends a random number of printable characters to buddy and exits
+void generator(event_based_actor* self, actor buddy) {
```

Comment:
A `generator` having a buddy gives little insight into what this relation actually is. Did you consider naming it something like "sink" to make this relation obvious on first read?

```
// Sends a random number of printable characters to `sink` and exits.
void generator(event_based_actor* self, actor sink) {
```

Reply:
Makes more sense.